app/react/render.js

```diff
@@ -0,0 +1,7 @@
+const React = require('react');
+
+module.exports = {
+  renderStory(Story) {
```

Comment:
What if I want to add extra args? Consider the following example:

```js
import { render, fireEvent } from '@testing-library/react';
import { Text } from './Button.stories';
it('should respond to click events', () => {
  const handleClick = jest.fn();
  const instance = render(
    <Text {...Text.args} onClick={handleClick} />
  );
  fireEvent.click(instance.container.firstChild);
  expect(handleClick).toHaveBeenCalled();
});
```

Reply:
do you think it should be `renderStory(Story, args?)`?

Reply:
that makes more sense to me